src/pages/index.js

```diff
@@ -135,12 +136,14 @@ const useStyles = makeStyles((theme) => ({
   },
   notification: {
     marginTop: theme.spacing(0.5),
+    marginBottom: theme.spacing(1),
```

Comment:
To support more than 2 notifications, proably better to make this `theme.spacing(0.5)` and add this class to your notification?

Reply:
Never mind, I see that component doesn't support a className prop. Ignore, and we can fix it later as needed.